crates/rnote-ui/src/dialogs/export.rs

```diff
@@ -223,7 +223,13 @@ pub(crate) async fn dialog_export_doc_w_prefs(appwindow: &RnAppWindow, canvas: &
                                 log::error!("exporting document failed, Error: `{e:?}`");
                                 appwindow.overlays().dispatch_toast_error(&gettext("Exporting document failed"));
                             } else {
-                                appwindow.overlays().dispatch_toast_text(&gettext("Exported document successfully"), crate::overlays::TEXT_TOAST_TIMEOUT_DEFAULT);
+                                appwindow.overlays().dispatch_toast_w_button_singleton(&gettext("Exported document successfully"), &gettext("View in file manager"), clone!(@weak canvas, @weak appwindow => move |_reload_toast| {
+                                    let folder_path = file.parent().expect("the file should always have a parent directory").path().unwrap();
```

Comment:
I'd change the `expect` here to error handling. It might be unlikely that there is no parent, but we've had issues with the file picker in the past so I don't trust it to always do the right thing and in that case the app shouldn't crash.

Reply:
Fixed by newest commit